Index: src/log.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/log.py b/src/log.py
new file mode 100644
--- /dev/null	(date 1731433871192)
+++ b/src/log.py	(date 1731433871192)
@@ -0,0 +1,13 @@
+from logging import getLogger, basicConfig, DEBUG, StreamHandler, Formatter
+
+
+class Log:
+    console_handler = StreamHandler()
+    console_handler.setLevel(DEBUG)  # Устанавливаем уровень для консольного обработчика
+    formatter = Formatter("[%(asctime)s] %(levelname)s - %(message)s")
+    console_handler.setFormatter(formatter)
+
+    basicConfig(level=DEBUG, handlers=[console_handler])
+
+    logger = getLogger(__name__)
+    logger.debug("This is a debug message.")
\ No newline at end of file
